easybuild/easyconfigs/p/ParMETIS/ParMETIS-4.0.3-intel-2018b-64bit.eb

```diff
@@ -0,0 +1,28 @@
+name = 'ParMETIS'
+version = '4.0.3'
+versionsuffix = '-64bit'
+
+homepage = 'http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview'
+description = """ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning
+ unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the
+ functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and
+ large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way
+ graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis/OLD',
+]
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['ParMETIS-4.0.3_64bit.patch']
```

Comment:
@smoors Please change to:

```python
patches = ['ParMETIS-%(version)s_64bit.patch']
```

Reply:
@boegel I will change it, but I disagree: it increases overhead because in the next version we will have to put the literal string '4.0.3' back.

Reply:
It serves as a reminder that the patch may have to be updated for newer software versions.